doc/source/installation.rst

```diff
@@ -37,23 +37,26 @@ Following guides describe the installation process for using the official Percon
 Installing |Percona Server| from a Binary Tarball
 ===================================================
 
-In |Percona Server| 5.7.31-34 and later, the multiple binary tarballs have been replaced with the following:
+In |Percona Server| 8.0.20-11 and later, the multiple binary tarballs have been replaced with the following:
 
-.. tabularcolumns:: |p{5cm}|p{11cm}|
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{11cm}|
 
 .. list-table::
    :header-rows: 1
 
-   * - Name
+   * - Type
+     - Name
      - Description
-   * - Percona-Server-5.7.xx-xx-Linux.x86_64.glibc2.12.tar.gz 
+   * - Full
+     - Percona-Server-<version number>-Linux.x86_64.glibc2.12.tar.gz 
      - Contains binaries, libraries, test files, and debug symbols
-   * - Percona-Server-5.7.xx-xx-Linux.x86_64.glibc2.12-minimal.tar.gz
+   * - Minimal
+     - Percona-Server-<version number>-Linux.x86_64.glibc2.12-minimal.tar.gz
      - Contains binaries, and libraries but does not include test files, or debug symbols
 
-Both binary tarballs support all distributions.
+Select the |Percona Server| 8.0 version number and type of tarball for your installation. Both binary tarballs support all distributions.
 
-|Percona Server| offers multiple tarballs depending on the *OpenSSL* library available in the distribution:
+In |Percona Server| before 8.0.20-11, multiple tarballs are provided based on the glibc version available in the distribution:
```

Comment:
Sorry I was wrong I missed key word ‘before 8.0.20’ yes before 8.0.20 we provided tars based on OpenSSL version. Please revert this change back